dashboard/src/components/Home.vue

```diff
@@ -35,7 +35,7 @@ export default {
 };
 </script>
 <style scoped lang="scss">
-.landing {
+.Home {
```

Comment:
Once we have a more final version of the site, I will make a pass at all of the higher-level class names.
Generally when working with components, they should have the same name.